src/components/body/components/sideBar/components/sideBarHeader/SideBarHeader.tsx

```diff
@@ -0,0 +1,29 @@
+//components
+import { Avatar } from "../../../../../avatar/Avatar";
+
+//style
+import styles from "./SideBarHeader.module.css";
+
+type SideBarHeaderProps = {
+  companyLogo: string;
+  height: string;
+  companyName: string;
+};
+
+export const SideBarHeader = ({
+  companyLogo,
+  height,
+  companyName,
```

Comment:
Fix prop names.
companyLogo <> imgSrc
companyName <> title

No need to pass height as a prop.